core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -148,7 +156,7 @@ var ExplorationEditorSettingsTab = function() {
       'Exploration Objective input', explorationObjectiveInput);
     await action.sendKeys(
       'Exploration Objective input', explorationObjectiveInput, objective);
-    await action.click('Neutral element', neutralElement);
+    await action.click('Neutral element', neutralElement, true);
```

Comment:
there are still issues with clicking on masked elements, for example here

Reply:
removed